### PR TITLE
Success page for Near Shell login (#570)

### PR DESCRIPTION
**Pull request by marcinbodnar**

### CODE REVIEW

**Comment by vgrichina** [comment]
@kcole16 @marcinbodnar this more a meta-common on https://github.com/near/near-wallet/issues/570 than this PR in particular:
- I think we actually need to figure out root reasons for problems on Windows/Linux before patching away. Pretty sure it doesn't have to be smth big, e.g. in case of GitPod it was just wrong type of redirect: https://github.com/near/near-shell/pull/363
- We can get both of best worlds if we provide a landing page in wallet (like done here), but also make a way for it to call back localhost (if port was opened successfully). We probably can just try loading 1x1 gif from certain URL or smth like that.

Basically need a bit more time to understand what's best way to integrate this into near-shell experience.

**Comment by marcinbodnar** [comment]
@vgrichina what is the root reason here, it's about this thread?
https://near-protocol.slack.com/archives/C012NUGRDQ9/p1588765049028700

**Comment by vgrichina** [comment]
@marcinbodnar 
> @vgrichina what is the root reason here, it's about this thread?
> https://near-protocol.slack.com/archives/C012NUGRDQ9/p1588765049028700

root reason for the thread has been fixed here https://github.com/near/near-shell/pull/363 AFAIK

however we also have different issue where if somebody runs near-shell on remote machine – redirect after login won't work. I guess this change is for that case? However it will still need corresponding near-shell change.

Looks like we can merge this safely though and integrate with near-shell later.
